readme.md

```diff
@@ -1,4 +1,30 @@
-# Ignite CLI
+<div align="center">
+  <h1> Ignite </h1>
```

Comment:
Is it expected to just be `Ignite` now instead of `Ignite CLI`?

Reply:
Just Ignite